specification/trafficmanager/resource-manager/Microsoft.Network/2017-09-01-preview/trafficmanageranalytics.json

```diff
@@ -430,30 +427,32 @@
       ],
       "description": "Class representing a Traffic Manager HeatMap."
     },
-    "TrafficManagerUserMetricsKeyProperties": {
-       "properties": {
-         "key": {
-           "type" : "string",
-           "description": "The key returned by the Realtime User Metrics operation."
-         }
-       },
-       "description": "Class representing the properties of a Real User Metrics key operation response."
-     },
-     "TrafficManagerUserMetricsKeyModel": {
-       "properties": {
-         "properties": {
-           "x-ms-client-flatten": true,
-           "$ref": "#/definitions/TrafficManagerUserMetricsKeyProperties",
-           "description": "The key property returned from a Realtime User Metrics GET operation."
-         }
-       },
-       "allOf": [
-         {
-           "$ref": "#/definitions/ProxyResource"
-         }
+    "TrafficManagerUserMetricsKeyModel": {
+      "properties": {
+        "id": {
+          "type" : "string",
+          "description": "Fully qualified resource Id for the resource. Ex - /providers/Microsoft.Network/trafficManagerUserMetricsKeys/default"
```

Comment:
It seems you removed "default" evrywhere else, is the description still accurate?

Reply:
You're right, I think I should change this here. The "/default" is still returned in this description, though it shouldn't be; the call with "/default" on the end fails. I'll remove it here, and also in the API operation.